### PR TITLE
UuidHolder: doesn't recreate 'id' each minute

### PR DESCRIPTION
**Pull request by RingerJK**

### CODE REVIEW

File: MapboxVision/src/main/java/com/mapbox/vision/utils/UuidHolder.kt [hunk 1]
```diff
@@ -22,18 +21,15 @@ internal interface UuidHolder {
...
 
         override fun start() {
-            val backgroundTimestamp = prefs.getLong(BACKGROUND_TIMESTAMP_KEY, 0L)
-            val currentTimestamp = System.currentTimeMillis()
-            if (currentTimestamp - backgroundTimestamp > MAX_BACKGROUND_TO_REFRESH_UUID_MILLIS) {
-                uniqueId = UUID.randomUUID().toString()
-                prefs.edit().putString(UUID_KEY, uniqueId).apply()
-            } else {
-                uniqueId = prefs.getString(UUID_KEY, "")!!
+            if (uniqueId.isBlank()) {
```
Comment (yunikkk) [inline MapboxVision/src/main/java/com/mapbox/vision/utils/UuidHolder.kt]: Do you think we could get rid of start-stop functions and just have single property that inits UUID and stores it to prefs?

Comment (RingerJK) [reply MapboxVision/src/main/java/com/mapbox/vision/utils/UuidHolder.kt]: looks like it's architecture approach. Components are started all all together and stopped as well. No?

**Comment by RingerJK** [comment]
@yunikkk PTAL

**Comment by yunikkk** [comment]
Should TelemetrySyncManager be updated too?

**Comment by RingerJK** [comment]
@yunikkk good shot :) done

**Pull request merged**